projects/VitalRecord/IJE.cs

```diff
@@ -120,7 +120,7 @@ protected string NumericAllowingUnknown_Get(string ijeFieldName, string fhirFiel
         protected void NumericAllowingUnknown_Set(string ijeFieldName, string fhirFieldName, string value)
```

Comment:
From Saul's comment, I thought the all 8s logic should only apply to certain numeric fields. We use NumericAllowingUnknown_Set for a lot of fields. Should there be a new helper that's specifically for the all 8s fields? like NumericAllowingUnknownAndAll8s_Set or something along those lines

Reply:
I don't think vrdr has any properties with all 8s so this other helper should be within the BFDR project instead of within VitalRecord

Reply:
Updated the "8"s to match the IJE spreadsheet, see the PR description and comments in code.